testdata/discover-releases-scenario-1-stable-4.16.json

```diff
@@ -0,0 +1,11 @@
+{
+  "nodes": [
+    {
+      "version": "4.16.2",
```

Comment:
I'm not opposed to defensive testing, but there will never (if we can help it) be a production response from `https://api.openshift.com/api/upgrades_info/graph` where 4.16.2 is in `stable-4.16` but not in `fast-4.16`.  Because [`fast-4.16` means "it's GA"][1], while [`stable-4.16` means "it's GA, and we also waited some additional delay"][2].  The former set will always include all of the latter.

[1]: https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/updating_clusters/understanding-openshift-updates-1#understanding-update-channels-releases
[2]: https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/updating_clusters/understanding-openshift-updates-1#stable-version-channel_understanding-update-channels-releases

Reply:
ah, I was just testing if versions can be retrieved from different channels. If it makes more sense, I can rename one of the channels to `candidate-`.